module/VuFindConsole/tests/unit-tests/src/VuFindTest/Command/ScheduledSearch/NotifyCommandTest.php

```diff
@@ -139,55 +143,26 @@ public function testNotificationWithRecentExecution()
         $this->assertEquals(0, $commandTester->getStatusCode());
     }
 
-    /**
-     * Test behavior when notifications are waiting to be sent but user does not
-     * exist.
-     *
-     * @return void
-     */
-    public function testNotificationsWithMissingUser()
```

Comment:
Due to changes in user handling in the command, this test no longer applies, so I simply deleted it.